css/style.css

```diff
@@ -0,0 +1,203 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+header {
+  background-image: url(../img/hero-image.jpg);
+  background-position: 5px -744px;
+  background-repeat: no-repeat;
+  width: 1450px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+input {
+  width: 100px;
+  height: 19px;
+  padding: 5px 5px 5px 5px;
+  background: url(../img/search-icon.svg), #707070;
+  background-repeat: no-repeat, no-repeat;
+  background-position: 2px 2px;
+  border-radius: 12px;
+}
+
+input:focus {
+  width: 140px;
+  padding-left: 20px;
+
+  /* margin: 8px 9px; */
```

Comment:
Please remove all commented code from your HTML and CSS files.

Reply:
done